src/pages/SolverPage.tsx

```diff
@@ -85,6 +81,10 @@ export const SolverPage = () => {
           />
         ))}
       </div>
+      <div style={{ width: '100%' }}>

```

Comment:
Would avoid inline styles and use modules as much as possible, except if it's to add dynamic styling. Just so we don't mix it for clarity.

Reply:
totally, once I _actually_ prepare a PR for merge, I always go through it myself first and clean up all this junk :D

Reply:
Gotcha! My bad, I thought it was open for feedback already 🙈 Just gimme a shout when you want to merge it and I'll take another look 🙏